src/analytics/logWindow/useLogWindowAnalytics.ts

```diff
@@ -9,6 +9,10 @@ type Action =
   | { name: "Added Filter"; filterExpression: string }
   | { name: "Deleted Filter"; filterExpression: string }
   | { name: "Toggled Filter"; visible: boolean }
+  | { name: "Added Bookmark" }
+  | { name: "Navigated With Bookmark" }
+  | { name: "Deleted Bookmark" }
```

Comment:
`Removed Bookmark` imo is better phrasing, "delete" sort of implies to me that the bookmark entered the void never to be seen again. "remove" implies to me that the bookmark could appear again in the future